redash/models/__init__.py

```diff
@@ -322,41 +319,6 @@ def store_result(cls, org, data_source, query_hash, query, data, run_time, retri
     def groups(self):
         return self.data_source.groups
 
-    def make_csv_content(self):
```

Comment:
Used the opportunity to move this serialization logic from `redash.models` to `redash.serializers`.

Reply:
Great! and +1 for tests!